pkg/validation/exception/validate.go

```diff
@@ -14,13 +13,14 @@ const (
 )
 
 // Validate checks policy exception is valid
-func Validate(ctx context.Context, logger logr.Logger, polex *kyvernov2alpha1.PolicyException, po webhooks.ExceptionOptions) (error, []string) {
+func Validate(ctx context.Context, logger logr.Logger, polex *kyvernov2alpha1.PolicyException,
```

Comment:
@eddycharly how about passing the options from the struct

Reply:
Why don't you move it to `pkg/validation/exception` ? Something like `ValidationOptions` ?

Reply:
sounds good

Reply:
@eddycharly done